reference/platform-app-yaml.md

```diff
@@ -187,10 +187,10 @@ The `locations` key allows you to provide specific parameters for different URL
 
 It has a few subkeys, which are:
 
-* `root`: The folder to serve static assets for this location from, relative to the application root. Typically `public` or `web`.
-* `passthru`: Whether to forward disallowed and missing resources from this location to the application. Can be `true` or `false` (for an application capable of handling HTTP traffic directly) or a URI path string (for a CGI based application, e.g. PHP). Typically your application's front controller, such as `/index.php` or `/app.php`.
-* `index`: The file or files to consider when serving a request for a directory. Can be a file name, an array of file names or *null*. Typically `index.html`. Note that in order for this to work, the static file(s) should be allowed by your rules. For example, to use a file named `index.html` as an index file, your rules must allow elements that match the filename, like `- \.html$`.
-* `expires`: How long to allow static assets from this location to be cached (this affects the Cache-Control and Expires headers). Can be a time or *-1* for no caching. Times can be suffixed with "ms" (milliseconds), "s" (seconds), "m" (minutes), "h" (hours), "d" (days), "w" (weeks), "M" (months, 30d) or "y" (years, 365d). The `expires` directive and resulting headers are left out entirely if this isn't set.
+* `root`: The folder to serve static assets for this location from relative to the application root. The application root is the directory in which the `.platform.app.yaml` file is located.  Typical values for this property include `public` or `web`.  Setting it to `"/"` is not recommended, and its behavior may vary depending on the type of application.
+* `passthru`: Whether to forward disallowed and missing resources from this location to the application. Can be true, false or a URI path string relative to the web root. For non-PHP applications it will generally be just true or false.  In a PHP application, this will typically be the front controller such as `/index.php` or `/app.php`.  If your application relies on a `.htaccess` file with `mod_rewrite` under Apache, this entry is the equivalent. Note: The value of `passthru` must be an absolute URL path, which means if it does not begin with the same value as the location key it is under the passthru may evaluate to another entry. That may be useful when you want different cache settings for different paths, for instance, but want missing files in all of them to map back to the same front controller.
```

Comment:
What do you recommend instead of "equivalent"?  I want to try and give context here for the many PHP apps that use a mod_rewrite-based front controller, for which this is the way they'd do the same thing. It's not as robust as mod_rewrite, certainly, but it's the equivalent technique.  How can this be phrased better?